archlinuxcn/lib32-llvm-git/PKGBUILD

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/bash
-
+#Maintainer: Solomon Choina <shlomochoina@gmail.com>
 # Based on LordHeavy's LLVM
 pkgbase=lib32-llvm-git
 pkgname=('lib32-llvm-libs-git' 'lib32-clang-git' 'lib32-llvm-git')
 pkgdesc='Low Level Virtual Machine (git version)'
 pkgver=19.0.0_r492195.9b6bd7093ccb0
 pkgrel=1
 groups=('chaotic-mesa-git')
-arch=('x86_64')
+arch=($CARCH)
 url="https://llvm.org/"
-license=('custom:Apache 2.0 with LLVM Exception')
+license=('Apache-2.0 WITH LLVM-exception')
 makedepends=('git' 'cmake' 'ninja' 'python'
              'lib32-gcc-libs' 'lib32-libffi' 'lib32-libunwind'
-             'lib32-libxml2' 'lib32-zlib' 'lib32-zstd')
+             'lib32-libxml2' 'lib32-zlib' 'lib32-zstd' 'gcc-multilib' 'llvm-git')
 
 source=("llvm-project::git+https://github.com/llvm/llvm-project.git")
```

Comment:
Would it be possible for you to specify a branch or tag to minimize unnecessary file downloads？

Reply:
What’s the unnecessary download?

Reply:
There is no need to download other branches.

Reply:
It seems that adding a branch would not make any difference. [The vcs source implementation since at least 8 years ago](https://gitlab.archlinux.org/pacman/pacman/-/commit/3d4529335c598e79b5a483fedc4c9d5c12ef10f1#95bc5c775e8d07b78e981f7a5f6afe05c502a680_0_41) (probably the first version) has been doing a full clone no matter if branches or any references is provided. Also, the build machine caches the download so I doubt it'll make any difference anyway.